src/components/cylc/gscan/index.js

```diff
@@ -31,10 +31,6 @@ function getWorkflowSummary (workflow) {
       // a task in waiting, may not have any jobs
       if (taskProxy.jobs) {
         for (const job of taskProxy.jobs) {
-          // TODO: temporary fix, as the backend is sending ready jobs, but they will change in cylc flow&uiserver in the future
-          if (job.state === 'ready') {
-            continue
-          }
```

Comment:
(Not sure what this block was for?)

Reply:
I think we had jobs with state `ready`, but only tasks could have this state? I remember asking something about that in Element or GH, and then was told that that was not correct for jobs (if I recall correctly), and that there was a ticket somewhere about improvements for states...

I think this is not preventing a JavaScript error, but preventing `Job` components of being created with the `state=ready` in the UI.

Reply:
Ah, OK, we'll have to look to see if that is still a problem (which now will be, are there jobs with the "preparing" state, which is equally wrong).